snmp/datadog_checks/snmp/snmp.py

```diff
@@ -362,7 +366,11 @@ def _start_discovery(self):
 
     def check(self, instance):
         # type: (Dict[str, Any]) -> None
+        start_time = time.time()
+        self._submitted_metrics = 0
+        self.telemetry_tags = self._config.tags[:]
```

Comment:
considering line 439 `tags.extend(config.tags)`, do you need to init `telemetry_tags` to config tags? Isn't this adding twice config tags?

Reply:
@pducolin You are right.

I refactored this part, should be ok now.